packages/components/src/button/index.tsx

```diff
@@ -17,8 +17,18 @@ import { useInstanceId } from '@wordpress/compose';
 import Tooltip from '../tooltip';
 import Icon from '../icon';
 import { VisuallyHidden } from '../visually-hidden';
+import type { WordPressComponentProps } from '../ui/context';
+import type {
+	ButtonProps,
+	DeprecatedButtonProps,
+	DisabledEvent,
+	TagName,
+} from './types';
 
-const disabledEventsOnDisabledButton = [ 'onMouseDown', 'onClick' ];
+const disabledEventsOnDisabledButton: DisabledEvent[] = [
+	'onMouseDown',
+	'onClick',
+];
```

Comment:
```suggestion
const disabledEventsOnDisabledButton = [
	'onMouseDown',
	'onClick',
] as const;
```

We can just use `as const` here instead of declaring the type separately. It tells TS to lock the array from mutation, which then gives it permission to treat it like an enum.

Reply:
That's a nice one! Committed in https://github.com/WordPress/gutenberg/pull/46997/commits/309db5cf29dc7dd2cc9291841223c53936a8443d#diff-d7a7249cdd571803b98fdd09650036566a355a402a68a58043a018acf0970ddbR28.

You showed me that before, sorry for missing that this time.